src/plugins/condition/utils/operations.js

```diff
@@ -85,10 +90,14 @@ export const OPERATIONS = [
             return ' is between ' + values.join(', ') + ' and ' + values[1];
         }
     },
-    {
+    {//45 is not between 100 and 89
```

Comment:
Let's remove this comment

Reply:
Sorry about that. I'll be more diligent about removing these going forward.